stdlib/fcntl.pyi

```diff
@@ -123,6 +119,18 @@ if sys.platform != "win32":
         RWH_WRITE_LIFE_LONG: Final = 4
         RWH_WRITE_LIFE_EXTREME: Final = 5
 
+    if (sys.version_info >= (3, 9) and sys.platform == "linux") or (sys.version_info >= (3, 13) and sys.platform == "darwin"):
```

Comment:
On my local I can access these on 3.11+, but stubtest fails if we don't have this 3.13+, not sure if anyone has an idea what might be going on. This is at least an improvement in the interim.